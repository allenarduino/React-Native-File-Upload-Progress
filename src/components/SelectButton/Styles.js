import { Constants } from '../../config/Constants';
export const Styles = {
  Button: {
    borderColor: Constants.borderColor,
    borderWidth: Constants.borderWidth,
    width: 105,
    padding: (8, 8),
    marginTop: 40,
    borderRadius: Constants.borderRadius,
  },

  ButtonText: {
    color: Constants.buttonTextColor,
    alignSelf: 'center',
    fontSize: Constants.fontSize,
    fontFamily: Constants.fontFamily,
  },
};
