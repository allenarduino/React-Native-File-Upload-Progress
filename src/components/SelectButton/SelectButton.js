import React from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { Styles } from './Styles';

export const SelectButton = ({ handleChoosePhoto }) => {
  return (
    <TouchableOpacity style={Styles.Button} onPress={handleChoosePhoto}>
      <Text style={Styles.ButtonText}>Select File</Text>
    </TouchableOpacity>
  );
};
