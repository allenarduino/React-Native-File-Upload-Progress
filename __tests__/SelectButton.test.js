import React from 'react';
import renderer from 'react-test-renderer';
import { SelectButton } from '../src/components/SelectButton/SelectButton';

describe('< Select Button />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<SelectButton />).toJSON();
    expect(tree.children.length).toBe(1);
  });
});

it('renders correctly on the screen', () => {
  const tree = renderer.create().toJSON();
  expect(tree).toMatchSnapshot();
});
