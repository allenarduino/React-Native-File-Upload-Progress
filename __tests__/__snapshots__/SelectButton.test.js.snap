// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly across screens 1`] = `null`;

exports[`renders correctly on the screen 1`] = `null`;
